reference/constraints/Callback.rst

```diff
@@ -178,6 +195,18 @@ You can then use the following configuration to invoke this validator:
         {
         }
 
+    .. code-block:: php-attributes
+
+        // src/Entity/Author.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        #[Assert\Callback(["Acme\Validator", "validate"])]
```

Comment:
```suggestion
        use Acme\Validator;
        use Symfony\Component\Validator\Constraints as Assert;

        #[Assert\Callback([Validator::class, 'validate'])]
```

* We should prefer `::class` over FQCN strings in PHP code.
* We should use single quotes if we don't need the additional semantics of double quotes.

Reply:
Ok, good to know. I have created attributes examples based on the annotation. I will fix it everywhere. In some other pull request, I will also update the annotations.